templates/home.html

```diff
@@ -5,6 +5,9 @@
   <main role="main" class="homepage">
     <div class="container">
 <!--       Home page content here -->
+      {{included.contents|safe}}
+      {{included.testKey|safe}}
+      {{included.sourcefile|safe}}
```

Comment:
Looked at the fork... Perhaps rather than having include use Markdown for everything, we might be able to use a `markdown` filter...

https://github.com/paularmstrong/swig-extras/blob/master/lib/filters/markdown.js
```json
{
  "metalsmith-swig-helpers": {
    "filters": {
      "markdown": "swig-extras/lib/filters/markdown"
    }
  }
}
```
``` html
{{included.contents|safe|markdown}}
{{included.testKey|safe|markdown}}
{{included.sourcefile|safe|markdown}}
```